core/templates/dev/head/pages/exploration_editor/ExplorationMetadataFormDirective.js

```diff
@@ -0,0 +1,146 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the exploration metadata form.
+ */
+oppia.directive('explorationMetadataForm', [
+ 'UrlInterpolationService',
```

Comment:
Similar to other directives, the indentation here should be by 2 spaces from the previous line. The same applies on line 34 and line 37 (and possibly others). Basically every new "scope" in our JS code is indented by 2 spaces from the previous line.